races/android.md

```diff
@@ -1,3 +1,87 @@
 ---
 wrapper: "../../../js/components/wiki/StubArticle"
 ---
```

Comment:
you should add a title and description
the title is important so the page name doesn't change if we have to rename the file
the description is just a fun tagline that shows up with links to your article